futures-executor/src/local_pool.rs

```diff
@@ -53,10 +54,14 @@ thread_local! {
     });
 }
 
-impl Wake for ThreadNotify {
+impl ArcWake for ThreadNotify {
     fn wake(arc_self: &Arc<Self>) {
         arc_self.thread.unpark();
     }
+
+    unsafe fn wake_local(arc_self: &Arc<Self>) {
+        Self::wake(arc_self);
```

Comment:
This is the same as the default impl-- no need to override

Reply:
Good point, I somehow missed this. Will fix it for all instances